netspeed/src/backend.c

```diff
@@ -149,35 +149,30 @@ free_devices_list(GList *list)
 /* Frees a DevInfo struct and all the stuff it contains
  */
 void
-free_device_info(DevInfo *devinfo)
+free_device_info (DevInfo *devinfo)
 {
-	g_free(devinfo->name);
-	g_free(devinfo->ip);
-	g_free(devinfo->netmask);
-	g_free(devinfo->ptpip);
-	g_free(devinfo->ipv6);
-	g_free(devinfo->essid);
+	g_free (devinfo->name);
+	g_free (devinfo->essid);
 #ifdef HAVE_NL
-	g_free(devinfo->tx_bitrate);
-	g_free(devinfo->rx_bitrate);
-	g_free(devinfo->channel);
+	g_free (devinfo->tx_bitrate);
+	g_free (devinfo->rx_bitrate);
+	g_free (devinfo->channel);
 #endif /* HAVE_NL */
+	g_free (devinfo);
 }
 
-static char*
-format_ipv4(guint32 ip)
+static void
+format_ipv4 (guint32  ip,
+             char    *dest)
 {
-	char *str = g_malloc(INET_ADDRSTRLEN);
-	inet_ntop(AF_INET, &ip, str, INET_ADDRSTRLEN);
-	return str;
+    inet_ntop (AF_INET, &ip, dest, INET_ADDRSTRLEN);
```

Comment:
the indentation is wrong

Reply:
No, it isn't.

Reply:
it doesn't respect old style

Reply:
As far as possible, on the new written code the tabs should be replaced by spaces.

Reply:
At my point of view it can be done in the whole file with one commit, if not we have one file with mixed tabs and spaces and the look is very weird

@raveit65 
what do you think about this?

Reply:
I respect your opinion, but I don't plan to format 1661 lines ;-)

Reply:
but you can respect the old style

(btw, there are tools to do the job)

Reply:
About using tools for formatting the code: After using such tools, the writers and the reviewers have to read, check and test it... fixing the mistakes, improving the automatic formatted code.... a hard job that can be broken down into small units by doing them only when new code is written.

Reply:
We have other open PRs of the applet. Rewriting the whole file will broke them and makes all complicated.
I think it better to do this later when all open PRs are finished.

I downloaded the raw patch https://patch-diff.githubusercontent.com/raw/mate-desktop/mate-applets/pull/476.diff
and found out that tab-size is 5 in this file!!!! Which is complete wrong, ihmo.
Change tab-size to 5 in pluma and the `diff` looks fine.
LGTM to use 4 spaces to start a new lin in newly written code.

This is a very good example how bad it is to use tabs in code for lazy reasons.